gridstatus/ercot.py

```diff
@@ -1773,6 +1777,49 @@ def _make_bid_curve(df):
 
         return self.parse_doc(df, verbose=verbose)
 
+    @support_date_range("DAY_START")
+    def get_sced_system_lambda(self, date, verbose=False):
+        """Get System lambda of each successful SCED
+
+        5 Minute Publish Interval
+
+        https://github.com/kmax12/gridstatus/issues/269
+
+        Arguments:
+            date (str, datetime): date to get data for
+            verbose (bool, optional): print verbose output. Defaults to False.
+
+        Returns:
+            pandas.DataFrame: A DataFrame
+
+        """
+
+        # just so we can share parse logic
+        kwargs = dict(
+            report_type_id=SCED_SYSTEM_LAMBDA_RTID,
+            date=date,
+            verbose=verbose,
+            extension="csv",
+        )
+
+        if date == "latest":
+            doc = self._get_document(**kwargs)
+            docs = [doc]
+        else:
+            docs = self._get_documents(**kwargs)
+
+        df = pd.concat([pd.read_csv(i.url, compression="zip") for i in docs])
```

Comment:
to make this easier to use with backfilling data that is sent over directly by ERCOT, can you separate out the logic for handling documents into a _handle_sced_system_lambda function? you can see an example of this here: https://github.com/kmax12/gridstatus/blob/main/gridstatus/ercot.py#L1889